diff --git a/build/withXcparse.js b/build/withXcparse.js
--- a/build/withXcparse.js
+++ b/build/withXcparse.js
@@ -27,6 +27,7 @@
 const xcode_1 = require("@bacons/xcode");
 const xcodeParse = __importStar(require("@bacons/xcode/json"));
 const config_plugins_1 = require("@expo/config-plugins");
+const createBaseMod_1 = require("@expo/config-plugins/build/plugins/createBaseMod");
 const fs = __importStar(require("fs"));
 const customModName = "xcodeProjectBeta2";
 const withXcodeProjectBeta = (config, action) => {
@@ -38,13 +39,13 @@
 };
 exports.withXcodeProjectBeta = withXcodeProjectBeta;
 const withXcodeProjectBetaBaseModInternal = (config) => {
-    return config_plugins_1.BaseMods.withGeneratedBaseMods(config, {
+    return (0, createBaseMod_1.withGeneratedBaseMods)(config, {
         platform: "ios",
         saveToInternal: true,
         skipEmptyMod: false,
         providers: {
             // Append a custom rule to supply AppDelegate header data to mods on `mods.ios.AppClipInfoPlist`
-            [customModName]: config_plugins_1.BaseMods.provider({
+            [customModName]: (0, createBaseMod_1.provider)({
                 isIntrospective: false,
                 // isIntrospective: true,
                 async getFilePath({ modRequest, _internal }) {
